cpp/tests/io/csv/csv_test.cu

```diff
@@ -42,6 +42,18 @@ bool checkFile(const char *fname)
     return (stat(fname, &st) ? 0 : 1);
 }
 
+char* getTempFile(const char *fname)
```

Comment:
I think you should create a fixture for these tests that creates the temp directory at startup and cleans up the directory at shutdown (after all tests).  All of the TEST() in this file will becomes TEST_F().

Reply:
Also, please rename this function `get_temp_filename`. We use underscore naming in libcudf, not camel case. Also, note "filename", not "file", since it doesn't return a file handle.

Reply:
```
The following tests FAILED:
	 19 - CSV_TEST (Failed)
	 31 - JSON_TEST (Failed)
```
The checks pass in my machine dgx13. but fails in CI.